package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/Wolox/eslint-config-wolox/issues"
   },
   "peerDependencies": {
-    "eslint": "5.9.0",
+    "eslint": "^6.4.0",
```

Comment:
We should be more open with peer dependencies, i think in lock only the major version:
```
"eslint": "6.x",
    "eslint-config-wolox": "3.x",
    "prettier": "1.x",
    "prettier-eslint": "9.x",
    "eslint-plugin-prettier": "3.x",
    "eslint-plugin-import": "2.x"
```